imagebuildah/build.go

```diff
@@ -1358,7 +1362,7 @@ func (b *Executor) Build(ctx context.Context, stages imagebuilder.Stages) (strin
 	ignoreLayers := singleLineDockerfile || !b.layers && !b.noCache
 
 	if ignoreLayers {
-		imgID, ref, err := stageExecutor.Commit(ctx, stages[len(stages)-1].Builder, "")
+		imgID, ref, err := stageExecutor.Commit(ctx, stages[len(stages)-1].Builder, "", b.output)
```

Comment:
should this and line 1348 just be 'output' and not 'b.output'?

Reply:
This one should be `b.output`, since it's committing the final image, but you're right that 1348 should be `output` rather than `b.output`, since except when it's the final stage, we don't want to be assigning the final image's name to the image that's committed there (though that should only happen when the final stage has an "AS" attached to it, which in any case would be weird).  Nice catch!  Fixed.